src/constants/queryParams.ts

```diff
@@ -0,0 +1,8 @@
+enum queryParams {
+  search = "search",
+  highlights = "highlights",
+  bookmarks = "bookmarks",
+  filters = "filters",
+}
+
+export { queryParams };
```

Comment:
Enums are typically capitalized I believe
```suggestion
enum QueryParams {
  Search = "search",
  Highlights = "highlights",
  Bookmarks = "bookmarks",
  Filters = "filters",
}

export { QueryParams };
```